interface/src/app/node/node.js

```diff
@@ -4,7 +4,9 @@
 
 angular.module('adage.node', [
   'ui.router',
-  'ui.bootstrap'
+  'ui.bootstrap',
+
```

Comment:
I like the way that angular's modules and ours are in different groups. But I know some coding guides (not Google's) don't like a blank line inside an array definition. I guess you can replace the blank line with a comment line if you want. It is totally up to you.